docs/getting-started/beginning-tutorial.md

```diff
@@ -1,12 +1,16 @@
 import Command from '@theme/Command'
 import Link from '@docusaurus/Link'
 
-# Your First Tauri App
+# Your First Node.js-based Tauri App
```

Comment:
I think Node.js based sets wrong expectations. People will assume the actual app is node based and never stop asking questions 😅

Maybe you can change the alert to something like: "The follow example uses a JavaScript build tool, so make sure to have Node.js and your favorite package manager installed."

Reply:
Mmmmh yeah, "Your First Node.js-based-frontend Tauri App" would be more accurate but perhaps overkill.
Okay, I'll remove this addition but we should specify that this tutorial is not the only way to write a Tauri app, maybe just the most common. We should point users to examples or make others guides: one without a bundler and perhaps one with a Rust frontend framework.

Reply:
> Mmmmh yeah, "Your First Node.js-based-frontend Tauri App" would be more accurate but perhaps overkill.
> 
> Okay, I'll remove this addition but we should specify that this tutorial is not the only way to write a Tauri app, maybe just the most common. We should point users to examples or make others guides: one without a bundler and perhaps one with a Rust frontend framework.

Sounds good to me!